.circleci/config.yml

```diff
@@ -424,3 +462,11 @@ workflows:
       - android-build:
           requires:
             - lint-testunit
+      - android-hold-google-play-alpha:
+          type: approval
+          requires:
+            - android-build
+      - upload-google-play-alpha:
+          requires:
+            - android-hold-google-play-alpha
+            
```

Comment:
It's weird for me.
Check how the `new line at end of file` looks at my files:
![Screen Shot 2020-06-03 at 19 42 24](https://user-images.githubusercontent.com/29778115/83696277-6dad2380-a5d2-11ea-82ae-0c18c918c86c.png)
And how it's looking here:
![Screen Shot 2020-06-03 at 19 43 03](https://user-images.githubusercontent.com/29778115/83696296-769df500-a5d2-11ea-9d22-0e6e29e75aa0.png)
I think maybe you did a mistake.

Reply:
Yes weird for me too, will look into it